core/src/proxy/block_engine_stage.rs

```diff
@@ -46,6 +46,7 @@ use {
 
 const CONNECTION_TIMEOUT_S: u64 = 10;
 const CONNECTION_BACKOFF_S: u64 = 5;
+const CONFIG_BACKOFF_S: u64 = 30;
```

Comment:
does this mean shutdown takes ~30s? probably wanna shorten it

Reply:
yeah, if it shuts down with an invalid config, it could take up to 30s, will shorten

if it has a non-empty url, it will exit immediately